support.sh

```diff
@@ -65,15 +65,17 @@ for networkID in $(${DOCKER} network ls --no-trunc --filter driver=overlay -q) "
 done
 
 echo "Container network configuration"
-for containerID in $(${DOCKER} container ls -q); do
+while read containerID status; do
     echo "ccc Container ${containerID}"
```

Comment:
would it be useful to put here what is the container state??
`ccc Container xxxxxxxx state: Up/Down/Failed etc`

Reply:
not to me, since that information is present in the `State` line, four lines later.

Reply:
is there a concern that suddenly logging information on stopped containers will trip up current post-processing?

Reply:
was thinking was just useful to have it at the top

Reply:
it's already there :D, I'm suggesting to just add the string in the header of the list

Reply:
good point. i'll add it.

Reply:
`state: Up/Down/Failed`:  with or without the whitespace after the colon?

Reply:
ccc Container xxxxxxxx state: Up
vs
ccc Container xxxxxxxx state:Up
vs
ccc Container xxxxxxxx state Up

I would vote for the first option, but 3rd option is in line with the fact that we don't have `:` for the container id

Reply:
Going with first option.  Rebased onto master.  Testing in support dump on orca branch `release/v3.0.x` before updating here.  Should be done with that shortly.